cmd/infra/powervs/create.go

```diff
@@ -101,13 +101,13 @@ var vpcDefaultUrl = func(region string) string { return fmt.Sprintf("https://%s.
 
 var log = func(name string) logr.Logger { return hypershiftLog.Log.WithName(name) }
 
-// MarshalJSON ...
+// MarshalJSON
 // custom marshaling func for time.Duration to parse Duration into user-friendly format
 func (d *TimeDuration) MarshalJSON() (b []byte, err error) {
 	return []byte(fmt.Sprintf(`"%s"`, d.Round(time.Millisecond).String())), nil
 }
 
-// UnmarshalJSON ...
+// UnmarshalJSON
```

Comment:
nit: Can we please stick to the convention we use everything else https://go.dev/blog/godoc? e.g
```
// UnmarshalJSON does custom unmarshalling func for time.Duration
```

Reply:
Modified the comments to follow this convention